driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverFactory.java

```diff
@@ -324,14 +323,6 @@ private WebDriver newWebDriver(
             newDriver = startNewWebDriverSession(desktopWebDriverRequest, capabilities, remoteAddress, sessionContext);
         }
 
-        /*
-        Log User Agent and executing host
-         */
-        if (remoteAddress != null && newDriver instanceof RemoteWebDriver) {
```

Comment:
See comment in `DesktopWebDriverUtils`.

Reply:
Please comment at telekom/testerra-selenoid-connector#6